### DISCUSSION TITLE
Is this a bug? Account not showing in dashboard.

### DISCUSSION THREAD

**Discussion by z0rg0n** [firefly-iii/firefly-iii#9231] [General]
If I make an account inactive it disappears from my dashboard.
If I then go back and make it active again, it does not show back up in the dashboard.

**Comment by JC5** [comment]
Hey great question. When an account is made inactive it's removed from the front page. Since you can pick and choose which accounts you want on the front page, it's not put back for you. You have to do it yourself in your preferences. Good luck!
